app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2 class="text-2xl text-blue-500 mb-16">Log in</h2>
+
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" ,class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 mt-4 rounded mb-10"%>
+  </div>
+<% end %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Name input field is missing on the page and there's error message popped while trying to register a user. Take a look.

###
![your-signup](https://github.com/shihabrafio/Blog_App/assets/26597432/3086777c-3c76-41fe-8dbc-6d6b9c1b52d4)

###
![your-signup-page-2](https://github.com/shihabrafio/Blog_App/assets/26597432/147c3422-47e8-476b-b303-677bd2c7ad42)